app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -755,8 +755,13 @@ class BrowserFragment : WebFragment(), LifecycleObserver, View.OnClickListener,
             Settings.getInstance(requireContext()).shouldUseBiometrics() &&
```

Comment:
(I can't drop a comment so far away from changed code.  Pretend this is on line 190)

If we split-screened between our app and settings, would `onResume` be called when moving focus between the two?  If not, since we've seen that `shouldUseBiometrics` can change during a session, we're opening ourselves up to a potential NPE on 190.  Suggest a null check there.   We're just turning off a listener, so we shouldn't have to do anything else to handle the case where the controller is null.